modular_skyrat/modules/departmentization/clothing_overrides.dm

```diff
@@ -1,7 +1,7 @@
 /obj/item/clothing/under/rank/cargo/tech
 	name = "cargo technician's jumpsuit"
-	desc = "A jumpsuit from the previous supply-department, a tag on the collar says, "Production Line: 2557, Product of Nanotrasen.""
```

Comment:
You can do `\"` if you want to be able to use quotes in text.

Reply:
![image](https://user-images.githubusercontent.com/62520989/212208572-9e9d2bb6-2f33-4c97-adcb-770f6edd0f91.png)